grr/server/grr_response_server/gui/ui/components/client_details/client_details.ng.html

```diff
@@ -0,0 +1,241 @@
+<div *ngIf="client$ | async as client" class="info">
+  <div class="header mat-body-1">
+    <mat-icon>computer</mat-icon>
+    <div class="mat-h1">{{client.knowledgeBase.fqdn}}
+      <div class="client-id mat-h3">{{client.clientId}}</div>
+    </div>
+  </div>
+  <div>
+    <div class="section-title mat-h3">General</div>
+    <table class="section">
```

Comment:
Using `<table>` is pretty infamous for non-tabular data nowadays. I haven't found a best practice on displaying lists of key-value data in Material Design though, so the table usage is good for now!

Reply:
I first looked into Material Design ways to do this, but there was nothing that would fit my needs, so I went old school with this one.